src/common/low_precision_transformations/src/lstm.cpp

```diff
@@ -0,0 +1,201 @@
+﻿// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "low_precision/lstm.hpp"
+
+#include <ngraph/pattern/op/wrap_type.hpp>
+#include <ngraph/opsets/opset1.hpp>
+
+#include <memory>
+#include <ngraph/node.hpp>
+#include <ngraph/opsets/opset1.hpp>
+#include <ngraph/opsets/opset5.hpp>
+#include <ngraph/pattern/op/or.hpp>
+
+#include "low_precision/concat.hpp"
+#include "low_precision/network_helper.hpp"
+#include "../include/low_precision/rt_info/skip_cleanup_attribute.hpp"
+
+namespace ngraph {
+namespace pass {
+namespace low_precision {
+
+LSTMTransformation::LSTMTransformation(const Params& params) : LayerTransformation(params) {
+    const auto X = ngraph::pattern::wrap_type<ngraph::opset5::Parameter>();
+    const auto H = ngraph::pattern::wrap_type<ngraph::opset5::Parameter>();
+    const auto C = ngraph::pattern::wrap_type<ngraph::opset5::Parameter>();
+    const auto W = ngraph::pattern::wrap_type<ngraph::opset5::Constant>();
+    const auto R = ngraph::pattern::wrap_type<ngraph::opset5::Constant>();
+    const auto B = ngraph::pattern::wrap_type<ngraph::opset5::Constant>();
+
+    const auto input_low = ngraph::pattern::wrap_type<ngraph::opset5::Constant>();
+    const auto input_high = ngraph::pattern::wrap_type<ngraph::opset5::Constant>();
+    const auto output_low = ngraph::pattern::wrap_type<ngraph::opset5::Constant>();
+    const auto output_high = ngraph::pattern::wrap_type<ngraph::opset5::Constant>();
+    const auto fq_X = ngraph::pattern::wrap_type<opset1::FakeQuantize>({ X,
+        input_low,
+        input_high,
+        output_low,
+        output_high});
+    const auto fq_H = ngraph::pattern::wrap_type<opset1::FakeQuantize>({ H,
+        input_low,
+        input_high,
+        output_low,
+        output_high });
+    const auto fq_W = ngraph::pattern::wrap_type<opset1::FakeQuantize>({ W,
+        input_low,
+        input_high,
+        output_low,
+        output_high });
+    const auto fq_R = ngraph::pattern::wrap_type<opset1::FakeQuantize>({ R,
+        input_low,
+        input_high,
+        output_low,
+        output_high });
```

Comment:
1) Don't use the same `Constant` operation instances for different `FakeQuantize`. You build function with four `FakeQuantize` with the same four `Constant`.
2) Create and reuse helper: `wrap_fake_quantize`

Reply:
done, thank you